packages/adapter-planetscale/src/planetscale.ts

```diff
@@ -146,7 +146,7 @@ class PlanetScaleTransaction extends PlanetScaleQueryable<planetScale.Transactio
 
 export class PrismaPlanetScale extends PlanetScaleQueryable<planetScale.Client> implements DriverAdapter {
   constructor(client: planetScale.Client) {
-    if (client['constructor']?.['name'] !== 'Client') {
+    if (!(client instanceof planetScale.Client)) {
```

Comment:
add a comment?

Since we don't have a test for the minification in this repo, it would help I think